### PR TITLE
docs(vital-demo-next, bodiless-webpack): Fix vital-demo-next docs

### PR DESCRIPTION
**Pull request by misterjones**
### Missing Docs in Vital Demo Next Site

Add packages to the `vital-demo-next` site's `package.json` file, filling in the gaps in the documentation.

See below for breakdown of added packages and related documentation sections/pages:

#### Vital Design System

- `@bodiless/vital-examples`
  - [Vital Design System > Curriculum](http://localhost:3000/#/VitalDesignSystem/Curriculum/)

#### Components

- `@bodiless/bv`
  - [Components > Bv](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/bv)
- `@bodiless/card`
  - [Components > Card](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Card)
- `@bodiless/carousel`
  - [Components > Carousel](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Carousel)
- `@bodiless/curator`
  - [Components > Social Wall](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/SocialWall)
- `@bodiless/ga4`
  - [Components > Analytics](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Analytics/)
- `@bodiless/navigation`
  - [Components > Navigation](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Navigation/)
- `@bodiless/oidc`
  - [Components > OIDC](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/OIDC)
- `@bodiless/organisms`
  - [Components > Embed](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Embed)
  - [Components > Filter By Group](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/FilterByGroup)
  - [Components > Single Accordion](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/SingleAccordion)
  - [Components > Social Share](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/SocialShare)
- `@bodiless/richtext`
  - [Components > Editors](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Editors/)
- `@bodiless/search`
  - [Components > Search](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Search/)
- `@bodiless/table`
  - [Components > Table](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Table)
- `@bodiless/youtube`
  - [Components > You Tube](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/YouTube)

#### Tools

- `@bodiless/accessibility`
  - [Tools > Accessibility](https://johnsonandjohnson.github.io/Bodiless-JS/#/Tools/Accessibility/)

#### Development

- `@bodiless/psh`
  - [Development > Web Platforms > Platform.sh](https://johnsonandjohnson.github.io/Bodiless-JS/#/Development/WebPlatforms/Platform.sh)
- `@bodiless/gatsby-theme-bodiless`
  - [Development > Gatsby Theme](https://johnsonandjohnson.github.io/Bodiless-JS/#/Development/GatsbyTheme)

### Remove Redundant Bodiless CLI Page

- Update `@bodiless/webpack` README file to no longer be an outdated clone of the "Bodiless CLI" page.
- Remove `@bodiless/webpack` README from being added to Docsify.
  - Remove Docsify-related files.
  - Remove `@bodiless/cli` from dependencies, as only used for Docsify.

----

## Test Instructions

01. Run the docs for the `vital-demo-next` site:
    `cd sites/vital-demo-next && npm run docs`
02. Go to <http://localhost:3000/#/>.
03. Verify that documentation matches "Live" documentation: <https://johnsonandjohnson.github.io/Bodiless-JS/#/>.

### CODE REVIEW

File: sites/vital-demo-next/package.json [hunk 1]
```diff
@@ -12,24 +12,39 @@
   },
   "dependencies": {
     "@babel/core": "^7.9.0",
+    "@bodiless/accessibility": "^1.0.0-rc.42",
```
Comment (hvanyo) [inline sites/vital-demo-next/package.json:15]: Purge -- going to depend on CI testing with their tool

File: sites/vital-demo-next/package.json [hunk 2]
```diff
@@ -12,24 +12,39 @@
...
     "@bodiless/backend": "^1.0.0-rc.42",
+    "@bodiless/bv": "^1.0.0-rc.42",
+    "@bodiless/card": "^1.0.0-rc.42",
```
Comment (hvanyo) [inline sites/vital-demo-next/package.json:18]: Purge -- old code and should deprecate.

File: sites/vital-demo-next/package.json [hunk 3]
```diff
@@ -12,24 +12,39 @@
...
+    "@bodiless/carousel": "^1.0.0-rc.42",
```
Comment (hvanyo) [inline sites/vital-demo-next/package.json:19]: Purge -- old code and should deprecate.

File: sites/vital-demo-next/package.json [hunk 4]
```diff
@@ -12,24 +12,39 @@
...
     "@bodiless/cli": "^1.0.0-rc.42",
     "@bodiless/components": "^1.0.0-rc.42",
     "@bodiless/core": "^1.0.0-rc.42",
+    "@bodiless/curator": "^1.0.0-rc.42",
     "@bodiless/documentation": "^1.0.0-rc.42",
     "@bodiless/fclasses": "^1.0.0-rc.42",
+    "@bodiless/ga4": "^1.0.0-rc.42",
+    "@bodiless/gatsby-theme-bodiless": "^1.0.0-rc.42",
```
Comment (hvanyo) [inline sites/vital-demo-next/package.json:27]: Purge -- Not relevant to nextjs

File: sites/vital-demo-next/package.json [hunk 5]
```diff
@@ -12,24 +12,39 @@
...
     "@bodiless/i18n": "^1.0.0-rc.42",
     "@bodiless/layouts": "^1.0.0-rc.42",
     "@bodiless/layouts-ui": "^1.0.0-rc.42",
+    "@bodiless/navigation": "^1.0.0-rc.42",
     "@bodiless/next": "^1.0.0-rc.42",
+    "@bodiless/oidc": "^1.0.0-rc.42",
```
Comment (hvanyo) [inline sites/vital-demo-next/package.json:33]: Purge -- until we use it.

File: sites/vital-demo-next/package.json [hunk 6]
```diff
@@ -12,24 +12,39 @@
...
+    "@bodiless/organisms": "^1.0.0-rc.42",
     "@bodiless/page": "^1.0.0-rc.42",
+    "@bodiless/psh": "^1.0.0-rc.42",
```
Comment (hvanyo) [inline sites/vital-demo-next/package.json:36]: hmm -- leave for now.  Not sure if we are using p.sh.

File: sites/vital-demo-next/package.json [hunk 7]
```diff
@@ -12,24 +12,39 @@
...
+    "@bodiless/richtext": "^1.0.0-rc.42",
     "@bodiless/schema-org": "^1.0.0-rc.42",
+    "@bodiless/search": "^1.0.0-rc.42",
```
Comment (hvanyo) [inline sites/vital-demo-next/package.json:39]: Purge we are not using this search

File: sites/vital-demo-next/package.json (hunk 1 above)
Comment (misterjones) [reply sites/vital-demo-next/package.json]: ✅ Purged.

File: sites/vital-demo-next/package.json (hunk 2 above)
Comment (misterjones) [reply sites/vital-demo-next/package.json]: ✅ Purged.

File: sites/vital-demo-next/package.json (hunk 3 above)
Comment (misterjones) [reply sites/vital-demo-next/package.json]: ✅ Purged.

File: sites/vital-demo-next/package.json (hunk 4 above)
Comment (misterjones) [reply sites/vital-demo-next/package.json]: ✅ Purged.

File: sites/vital-demo-next/package.json (hunk 6 above)
Comment (misterjones) [reply sites/vital-demo-next/package.json:31]: ✅ Leaving (for now).

File: sites/vital-demo-next/package.json (hunk 7 above)
Comment (misterjones) [reply sites/vital-demo-next/package.json]: ✅ Purged.

File: packages/bodiless-scaffold/package.json [hunk 8]
```diff
@@ -1,12 +1,11 @@
 {
-  "name": "@bodiless/vital-scaffold",
+  "name": "@bodiless/scaffold",
   "version": "1.0.0-rc.42",
   "description": "A scaffold tool to generate Bodiless-JS components and packages",
   "author": "Dewen Li <dli94@kenvue.com>",
   "homepage": "",
   "license": "Apache-2.0",
   "main": "./lib/index.js",
-  "type": "module",
```
Comment (hvanyo) [inline packages/bodiless-scaffold/package.json:9]: @dewen what is the impact of removing this?  It won't build docs when its there?

**Review by hvanyo — DISMISSED** [review]

**Review by dewen — APPROVED** [review]

File: packages/bodiless-scaffold/package.json (hunk 8 above)
Comment (dewen) [reply packages/bodiless-scaffold/package.json]: restored `type`, updated scaffold/getDocs.cjs instead

**Review by hvanyo — APPROVED** [review]

**Pull request merged**